python_cdk/src/detectors/python-cdk-elb-tls-https-listeners-only/python-cdk-elb-tls-https-listeners-only_non-compliant.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-elb-tls-https-listeners-only@v1.0 defects=1}
+import aws_cdk as cdk
+from aws_cdk import Stack, Vpc
+from aws_cdk.aws_elasticloadbalancing import LoadBalancer,LoadBalancingProtocol
+from aws_cdk import aws_elasticloadbalancing as elb
+
+class CdkStarterself(cdk.self):
+  def __init__(self, scope: cdk.App, id: str):
+    super(scope, id)
+    # Noncompliant: `LoadBalancingProtocol` is not set.
```

Comment:
"Noncompliant: The LoadBalancingProtocol is set to tcp for the internal protocol"